lib/topology_docker_openswitch/openswitch.py

```diff
@@ -121,9 +121,9 @@ def cur_cfg_is_set():
     sock.send(dumps(query))
     response = loads(sock.recv(4096))
     try:
-        return response['result'][0]['rows'][0]['cur_hw'] == 1
+        return response['result'][0]['rows'][0]['cur_hw']
```

Comment:
I added the verification in lines 193 and 194, I removed it so I can pass to the log output the obtained value